src/main/resources/api/service_openapi.yaml

```diff
@@ -702,6 +712,14 @@ components:
       items:
         $ref: '#/components/schemas/RoleBinding'
 
+    GrantRoleRequestBody:
+      description: The body for a request to grant a role to a single user.
+      type: object
+      required: [memberEmail]
+      properties:
+        memberEmail:
+          type: string
```

Comment:
A side benefit of using the body is that you don't have to escape the email address in the URL.

Reply:
+1, though I believe the jersey client and swagger-ui both handle that for us. still nice though